YAXLib/ReflectionUtils.cs

```diff
@@ -729,15 +730,17 @@ public static bool IsPublicProperty(PropertyInfo pi)
         public static bool IsPartOfNetFx(MemberInfo memberInfo)
         {
             var moduleName = memberInfo.Module.Name;
-#if NETSTANDARD || NET5_0
+
+            if (RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework"))
```

Comment:
```suggestion
            if (RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework"))
```
This modification comes from cross-framework compatibility. Guess it should not go into the initial YAXLib.Redux merge.

Reply:
Yea, cross-framework compatibility is the reason why `YAXLib.Redux` exist in the first place, :confused:
But if you think so then this will be implemented in a separated PR.